downloadinterceptor.cpp

```diff
@@ -105,3 +106,7 @@ void DownloadInterceptor::downloadFinished(QNetworkReply *reply) {
 
     reply->deleteLater();
 }
+
+void DownloadInterceptor::abort() {
+    emit abortDownload();
```

Comment:
@cibersheep couldn't you just store the reply as a variable of the DownloadInterceptor class (something like m_reply) and call m_reply->abort() ?

Reply:
Actually, doing it this way makes sense because it will cancel multiple downloads if that happens (makes things cleaner than only storing one reply).